### ISSUE TITLE
Integrate tidygate in `tidySPE`

### ISSUE THREAD

**Issue by stemangiola** [william-hutchison/tidySpatialExperiment#7]
tidygate allows to gate points from a tibble, visually procedurally and in a replicable way, compatibly with pipe-oriented programming style

https://github.com/stemangiola/tidygate

it would be beneficial to integrate it to `tidySpatialExperiment` and document it's use

an example of it's usage can be

```r

spe |> 
    mutate(region = tidygate(...)) |> 
    aggregate_cells(c(sample, region)) |> 
    tidybulk::test_differential_abundance(~ region + sample)

```

**Comment by william-hutchison** [comment]
This issue was solved with tidySpatialExperiment commit 9163279a40766ee30656bae1337bccd135fa9afe.

I have added documentation to tidySpatialExperiment demonstrating the use of tidygate. tidygate was already working well with tidySpatialExperiment, except pixel coordinate information from duplicate cells is dropped. 

Fixing this will be part of a broader project to get the entire tidyomics ecosystem to work with duplicate cell names.
